src/content/docs/en/reference/api-reference.mdx

```diff
@@ -687,6 +687,35 @@ export function GET(context: APIContext) {
 }
 ```
 
+### `Astro.getActionResult()`
+
```

Comment:
I'm not sure about the version, the changelog does not mention it. If I'm correct, this was added in https://github.com/withastro/astro/pull/10858 so from the linked release https://github.com/withastro/astro/pull/10944 I guess it was in `4.8.0`.

In `Type:`, I shortened the signature by omitting the type parameters to improve readability. I think this is enough to understand its usage.

```suggestion

<p>

**Type:** `(action: TAction) => ActionReturnType<TAction> | undefined`<br />
<Since v="4.8.0" />
</p>

```

Reply:
Fantastic! The important part is probably the `| undefined`, so this is great.